plugin/trino-faker/src/main/java/io/trino/plugin/faker/FakerPageSource.java

```diff
@@ -38,8 +38,10 @@
 import io.trino.spi.type.TimestampType;
 import io.trino.spi.type.TimestampWithTimeZoneType;
 import io.trino.spi.type.Type;
+import io.trino.spi.type.UuidType;
```

Comment:
Maybe make a separate PR for this commit ? This looks okay to merge already

Reply:
https://github.com/trinodb/trino/pull/24146